components/BlogPost.js

```diff
@@ -0,0 +1,13 @@
+import Link from "next/link";
+const BlogPost = ({ blogPost }) => {
```

Comment:
Instead of passing a single prop you can pass link, image, title as different props.